File: employment/employment/report/status/status.py
# Copyright (c) 2013, frappe and contributors
# For license information, please see license.txt

from __future__ import unicode_literals
# import frappe

def execute(filters=None):
	columns, data = [
                         {
                          "fieldname": "status",
                         "label": _("Status"),
                         "fieldtype": "Select",
                         "options": "Hold",
                         "width": 300
                        }

                      ], []
	return columns, data

File: employment/employment/report/status/status.js
// Copyright (c) 2016, frappe and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["Status"] = {
	"filters": [
               {

            "fieldname":"status",
            "label": __("Status"),
            "fieldtype": "Select",
            "options": "Hold",
            "default": frappe.defaults.get_user_default("Status")
               },

	]
};
